### PR TITLE
Adding fosscuda version of LAMMPS, with partial support to build on POWER9

### PR DESCRIPTION
**Pull request by edmondac**
(created using `eb --new-pr`)

Requires:

1. https://github.com/easybuilders/easybuild-easyblocks/pull/2254
1. https://github.com/easybuilders/easybuild-framework/pull/3506

There is a further problem building PLUMED on POWER9, where `asmjit` isn't supported AFAIK. But fixing that requires a custom easyblock for PLUMED, and I can't do that now!

There are a couple more tweaks needed to build on POWER9 - (see https://github.com/bear-rsg/easybuild-easyconfigs/pull/417 and related PRs for our local repo) - but they aren't in a form that would get merged here.

### CODE REVIEW

File: easybuild/easyconfigs/l/LAMMPS/LAMMPS-3Mar2020-fosscuda-2020a-Python-3.8.2-kokkos.eb [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+# SCAFACOS (listed below) will only be enabled if ScaFaCoS is a dependency (I.e. on x86_64 only)
+# YAFF(listed below) will only be enabled if yaff is a dependency (I.e. on x86_64 only)
+user_packages = [
+    'ATC', 'BOCS', 'CGDNA', 'CGSDK', 'COLVARS', 'DIFFRACTION', 'DPD', 'DRUDE',
+    'EFF', 'FEP', 'H5MD', 'LB', 'MANIFOLD', 'MEAMC', 'MESO', 'MGPT', 'MISC',
+    'MOFFF', 'MOLFILE', 'NETCDF', 'PHONON', 'PLUMED', 'PTM', 'QTB', 'REAXC',
+    'SCAFACOS', 'SMD', 'SMTBQ', 'SPH', 'TALLY', 'UEF', 'YAFF', 'VTK'
+]
+
+cuda_compute_capabilities = "3.5,5.2,6.0,7.0"
```
Comment (edmondac) [inline easybuild/easyconfigs/l/LAMMPS/LAMMPS-3Mar2020-fosscuda-2020a-Python-3.8.2-kokkos.eb:100]:
```suggestion
```

People can specify that on the comand line

File: easybuild/easyconfigs/p/PLUMED/PLUMED-2.6.0-fosscuda-2020a-Python-3.8.2.eb [hunk 2]
```diff
@@ -0,0 +1,60 @@
+# by Ward Poelmans <wpoely86@gmail.com>
```
Comment (edmondac) [inline easybuild/easyconfigs/p/PLUMED/PLUMED-2.6.0-fosscuda-2020a-Python-3.8.2.eb:1]:
```suggestion
# Copied from foss version which was by Ward Poelmans <wpoely86@gmail.com>
```

File: easybuild/easyconfigs/v/VTK/VTK-8.2.0-fosscuda-2020a-Python-3.8.2.eb [hunk 3]
```diff
@@ -0,0 +1,97 @@
+##
```
Comment (edmondac) [inline easybuild/easyconfigs/v/VTK/VTK-8.2.0-fosscuda-2020a-Python-3.8.2.eb:1]:
```suggestion
##
# Copied from foss version, which had the following message:
```

File: easybuild/easyconfigs/y/yaff/yaff-1.6.0-fosscuda-2020a-Python-3.8.2.eb [hunk 4]
```diff
@@ -0,0 +1,42 @@
+# Updated from previous config
```
Comment (edmondac) [inline easybuild/easyconfigs/y/yaff/yaff-1.6.0-fosscuda-2020a-Python-3.8.2.eb:1]:
```suggestion
# Copied from foss version, which had the following message:
# Updated from previous config
```

**Comment by branfosj** [comment]
As we (University of Birmingham) are no longer working on POWER9 support I am closing this. It would need some more changes to make it mergeable and anyone wishing to do so can work from there to do so.

**Pull request closed without merging**